src/org/armedbear/lisp/extensible-sequences.lisp

```diff
@@ -1036,4 +1036,5 @@ the sequence argument to be a proper sequence.
                   (tagbody
                      ,@forms))))))))))
 
-(provide "EXTENSIBLE-SEQUENCES")
\ No newline at end of file
+(provide "EXTENSIBLE-SEQUENCES")
+(push :extensible-sequences *features*)
```

Comment:
You might want a `pushnew` instead.

Reply:
Yes, `pushnew` would always be preferred.